src/screens/users/index.js

```diff
@@ -1,78 +1,90 @@
 import React, { PureComponent } from 'react';
-import { View, Text, TouchableOpacity } from '../../components';
-import { StyleSheet } from 'react-native'
+import { StyleSheet, TextInput } from 'react-native';
 import { connect } from 'react-redux';
+import { Text, View, Touchable } from '../../components';
+import * as COLORS from '../../constants/colors';
+import { search } from '../../store/users/actions'
 
 class Users extends PureComponent {
-    constructor() {
-        super();
-    }
-
     openChat = (receivingUser) => {
```

Comment:
Don't use brackets for one-liners.
  
`openChat = receivingUser => this.props.navigation.navigate('Chat', { receivingUser });`

Reply:
This is inconvenient for expanding the functionality, especially at the development stage.

Reply:
This is inconvenient for expanding the functionality, especially at the development stage.

Reply:
I don't think this function will expand. As i can see this function is only for navigation and you pass receivingUser as navigation param.

Reply:
And if you constantly write this is inconvenient at the development stage for expansion. I will look at refactoring later, especially, if it is a big project.

Reply:
This is not so critical for the performance) But it is convenient

Reply:
this is not first project)